webapp/controller/ProductDetailsPage.controller.js

```diff
@@ -91,6 +110,7 @@ sap.ui.define(
        */
       setDefaultViewModel: function () {
         const oDefaultProductDetailsViewModel = new JSONModel({
+          IsInCreateMode: false,
           IsInEditMode: false,
```

Comment:
Sometimes the variable name is a little overloaded, from my point of view IsEditMode or IsCreateMode is enough, this is just a note, there is no need to change it.